src/main/java/io/unlaunch/automation/pages/Registration.java

```diff
@@ -52,23 +52,23 @@ public void register() {
 
     public void otp() {
         Browser.fluentWait((WebDriver d) -> d.findElement(By.cssSelector("input[type=tel]"))).sendKeys("455148");
+        /* Verify otp right away after code is entered
```

Comment:
why is this commented out?

Reply:
because after code is entered, it verifies right away, there is no button to click.